src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/getImageBlock/index.js

```diff
@@ -1,4 +1,4 @@
-import htmlUnescape from '../../../../../utils/htmlUnescape';
+import htmlUnescape from '../../../../../../utils/htmlUnescape';
```

Comment:
Could we create a dir alias for this please? e.g. `#utils`

Reply:
@chris-hinds done!

Reply:
Thanks @rhenshaw56